gitpod-io/website
# [Analytics] Add context for tracking events sent from contact widget

## Description

Adds context to clicks on the contact widget by

- Adding a `{"context":"contact_widget"}` property to all clicks on the widget
- Manually adding labels to buttons that don't have any (Open / Close contact widget)

## How to test


1. Open Preview Environment
2. Go to Segment's [debugger of Website Staging](https://app.segment.com/gitpod/sources/website_staging/debugger)
3. Interact with the contact widget in the preview environment and ensure that the `website_clicked` events contain 1) a `context: "contact_widget"` property and 2) values != `null` in the label property for all clicks on the contact widget.

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2320"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>
